test/e2e/integreatly_test.go

```diff
@@ -27,14 +30,15 @@ import (
 
 var (
 	retryInterval                    = time.Second * 5
-	timeout                          = time.Second * 60
+	timeout                          = time.Second * 75
```

Comment:
this isn't really needed, and will remove in another PR so as not to trigger the e2e-spiral-of-doom :D